packages/server/src/adapters/ws.ts

```diff
@@ -140,7 +140,8 @@ export function applyWSSHandler<TRouter extends AnyRouter>(
       });
     }
 
-    const ctxPromise = createContext?.({ req, res: client });
+    // FIXME: info not set
+    const ctxPromise = createContext?.({ req, res: client, info: {} as any });
```

Comment:
I'm not very familiar with how subscriptions work here. Can we extract the request info here too in a nice way?

@KATT

Reply:
It's actually not sent here straight away... 

In subscriptions, we create a context _once_ when we open the connection and not as a result of a procedure.

I didn't consider this, but basically we have no further request info when doing websockets